portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/Policies.tsx

```diff
@@ -259,11 +307,11 @@ const Policies: React.FC = () => {
         } else {
             // Add new operation policy
             const uuid = uuidv4();
-            operationInAction.operationPolicies[currentFlow].push({ ...updatedOperation, uuid });
+            ((isAPILevelTabSelected) ? newApiLevelPolicies : operationInAction.operationPolicies)[currentFlow].push({ ...updatedOperation, uuid });
```

Comment:
Why have we used the name `newApiLevelPolicies`? What does the `new` imply here?